livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/realtime/realtime_model.py

```diff
@@ -541,7 +544,8 @@ def create(
                         message,
                         extra=self._sess.logging_extra(),
                     )
-                    return
+                    fut.set_result(False)
+                    return fut
```

Comment:
I know it's been like that before, but just thinking out loud, should we raise an Exception instead of logging a warning? Since the creation failed anyway

Reply:
What do you think if we let the user of this method to decide the agent to keep running or be failed based on the value of the Future.

Reply:
I think Python users don't tend to check the output of functions for errors. Most of the time they except seeing Exception and catching them

Reply:
Ok that makes sense, I can submit another PR for this.